### ISSUE TITLE
NAPS2 Dependency Error, Won't Install or Update, Causing Auto-Updates to Fail

### ISSUE THREAD

**Issue by AT-StephenDetomasi** [dgalbraith/chocolatey-packages#569]
This package, which has been working well, now appears to have a dependency error.

https://community.chocolatey.org/packages/naps2

I'm having some issues with this package, it seems to have some dependency issues that prevent it from installing or updating. The biggest issue for me is that it is actually preventing other packages from updating as well. Any ideas on when this issue will be resolved? Typically these dependencies should be handled by the package itself. For what it is worth, kb2919442 and dotnet-runtime already exist on my system.

```
PS C:\Users\Owner> choco install naps2
Chocolatey v2.2.2
3 validations performed. 2 success(es), 1 warning(s), and 0 error(s).

Validation Warnings:
 - System Cache directory is not locked down to administrators.
   Remove the directory 'C:\ProgramData\ChocolateyHttpCache' to have
   Chocolatey CLI create it with the proper permissions.

Installing the following packages:
naps2
By installing, you accept licenses for the packages.
[NuGet] One or more unresolved package dependency constraints detected in the Chocolatey lib folder. All dependency constraints must be resolved to add or update packages. If these packages are being updated this message may be ignored, if not the following error(s) may be blocking the current package operation: 'dotnet 8.0.4 constraint: dotnet-runtime (>= 8.0.4 && < 8.0.5)'
Unable to resolve dependency 'kb2919442': Unable to find a version of 'kb2919442' that is compatible with 'KB2919355 1.0.20160915 constraint: kb2919442 (>= 1.0.20160915)'.
```

**Comment by AT-StephenDetomasi** [comment]
Anyone? This breaks choco for me at the moment...

**Comment by dgalbraith** [comment]
I don't believe that this is an issue with the `naps2` package - it looks completely unrelated.

The full dependency chain for the package is  [naps2](https://community.chocolatey.org/packages/naps2) --> [naps2.install](https://community.chocolatey.org/packages/naps2.install) --> [dotnetfx (>= 4.8.0.0)](https://community.chocolatey.org/packages/dotnetfx) --> [chocolatey-dotnexfx.extension (>= 1.0.1)](https://community.chocolatey.org/packages/chocolatey-dotnetfx.extension).

There is no dependency on the packages for the security updates [kb2919442](https://community.chocolatey.org/packages/KB2919442) and  [kb2919355](https://community.chocolatey.org/packages/KB2919355) or the [dotnet-runtime (>= 8.0.4 && < 8.0.5)](https://community.chocolatey.org/packages/dotnet-runtime) that are referenced in the errors that you are seeing.

The dependency issues here don't look to be anything to do with the `naps2` package as far as I can see.
